### PR TITLE
Vinay login

### PR DESCRIPTION
**Pull request by Vinay-Sunny**
- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 AngularApp/node_modules
 TestFramework/node_modules
-ReactApp/node_modules
\ No newline at end of file
+ReactApp/node_modules
+ReactLatestApp/node_modules
```
Comment (khaziwallis) [inline .gitignore]: @Vinay-Sunny add new line at end of file.. so that we will not see the red arrow alert on github

File: ReactApp/src/index.js [hunk 2]
```diff
@@ -6,28 +6,31 @@
 */
 
 // react dependencies
-import React from 'react';
-import ReactDOM from 'react-dom';
+import React from "react";
```
Comment (khaziwallis) [inline ReactApp/src/index.js]: i guess, you need to revert all these ReactApp folder changes.

File: ReactLatestApp/src/App.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import { BrowserRouter, Route, Switch, Redirect } from "react-router-dom";
+import "./App.css";
+import decode from "jwt-decode";
+import LoginPage from "./components/login";
+import Home from "./components/home";
+import Error from "./components/error";
+
+// const checkAuth = () => {
```
Comment (khaziwallis) [inline ReactLatestApp/src/App.js]: @Vinay-Sunny you can remove all thee comments as well.. take up these changes as next PR.

**Comment by shanth21** [comment]
Dev LGTM

**Comment by shanth21** [comment]
DEv LGTM

**Pull request merged**